lib/constants.js

```diff
@@ -1,3 +1,3 @@
 export const SPARK_VERSION = '1.6.0'
-export const DELAY_BETWEEN_RETRIEVALS = 10_000
 export const MAX_CAR_SIZE = 200 * 1024 * 1024 // 200 MB
+export const APPROX_ROUND_LENGTH_IN_MS = 60 * 60_000 // 1 hour
```

Comment:
Let's keep this initial version simple.

In the future, we can query Meridian state to find out how many blocks remain in the current round and calculate the remaining round time using that value.